src/schemas.ts

```diff
@@ -167,7 +171,34 @@ export interface schema_i18nRegions_list extends param {
   part: string;
   hl?: string;
 }
+export interface schema_live_LiveBroadcasts_list extends param {
```

Comment:
I was actually thinking to have a separate file inside a `schemas/` directory to hold these (since I plan to support the analytics API as well in future), so if each API is independent, it'll be much cleaner.
The prefix `schemas_live` is a good choice, and you can go with that.